client/src/components/Pages/Help/index.js

```diff
@@ -26,11 +28,51 @@ const HelpPage = () => {
     ))
   );
 
+  const renderLinks = () => (
+    <ul className="shortcut-links-wrapper">
+      <li>
+        <a
+          className="waves-effect waves-light btn-large light-blue lighten-1"
+          href={STORYBOOK}
+        >
+          <i className="material-icons left">book</i>
+          Storybook
+        </a>
+      </li>
+      <li>
+        <a
+          className="waves-effect waves-light btn-large purple lighten-1"
+          href={STYLEGUIDE}
+        >
+          <i className="material-icons left">widgets</i>
+          Styleguidist
+        </a>
+      </li>
+      <li>
+        <Link
+          className="waves-effect waves-light btn-large teal lighten-1"
+          to={NOT_FOUND}
+        >
+          <i className="material-icons left">location_off</i>
+          404 page
+        </Link>
+      </li>
+    </ul>
+  );
+
   return (
     <div className="help-page-container">
       <h3 className="help-section-title">Help</h3>
       <p className="text">
-        If you are completely stuck and you have no idea what question
+        As this is a learning project, I want you to access easily to all the
+        hidden places of the project, even if normally users should not access them.
+        Im my case, they are: 404 page and documentations in
+        Sorybook and Styleguidist. It is something internal but maybe someone of you will like to
```

Comment:
`Will want` or `would like`, but I think `will like` is not the correct one... 😃